### DISCUSSION TITLE
Hetero conv question

### DISCUSSION THREAD

**Discussion by DogitoErgoSum** [pyg-team/pytorch_geometric#4482] [Q&A] [answered]
Imagine that i have 3 node types, A B and C. A and B have direct connections to C.
I will use HeteroConv as a wrapper for the convolutions and use different convolutions. Lets say i use GATConv for A to C  and use GINEConv for B to C, and lets say i will use the sum aggregation for everything.
Is x_i´ computed like this?
![image](https://user-images.githubusercontent.com/51344466/163581195-2848972c-8f9e-41a1-9eb9-1ceca2d4f3b3.png)

**Comment by rusty1s** [comment, marked as answer]
That is absolutely correct :)

**Reply by DogitoErgoSum** [reply]
Ty so much
